xrpl/models/utils.py

```diff
@@ -59,7 +87,11 @@ def new_init(self: _Self, *args: List[Any], **kwargs: Dict[str, Any]) -> None:
             )
         original_init(self, **kwargs)
 
-    # noinspection PyTypeHints
-    cls.__init__ = new_init  # type: ignore
-
+    # For Python v3.10 and above, the KW_ONLY attribute in data_class
+    # performs the functionality of require_kwargs_on_init class.
+    # When support for older versions of Python (earlier than v3.10) is removed, the
+    # usage of require_kwargs_on_init decorator on model classes can also be removed.
+    if is_kw_only_attr_defined_in_dataclass():
+        # noinspection PyTypeHints
```

Comment:
What is this comment for? I don't think PyTypeHints is something this library uses.

Reply:
Okay. I was trying to preserve the existing code. `#noinspection PyTypeHints` is also used in the present code base. I've removed it in the latest tip of this branch 👍